From: Simon McVittie <smcv@debian.org>
Date: Wed, 9 Sep 2020 10:03:31 +0100
Subject: tests: Use a more interoperable path to bash

On non-merged-/usr systems, bash is traditionally in /bin.
On merged-/usr systems, /bin and /usr/bin are synonymous so it doesn't
matter which one we use.
---
 tests/vnc-test-runner.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/vnc-test-runner.sh b/tests/vnc-test-runner.sh
index a699e74..eb62cd0 100755
--- a/tests/vnc-test-runner.sh
+++ b/tests/vnc-test-runner.sh
@@ -1,3 +1,3 @@
-#!/usr/bin/bash
+#!/bin/bash
 
 dbus-run-session -- xvfb-run -s '+iglx -noreset' $TEST_SRCDIR/tests/run-vnc-tests.py
